packages/manager/src/managers/telemetry/types.ts

```diff
@@ -35,13 +35,13 @@ export const SegmentEventType = {
 	sliceMachine_start: "slice-machine:start",
 	sliceLibrary_beta_modalOpened: "slice-library:beta:modal-opened",
 	sliceLibrary_beta_codeOpened: "slice-library:beta:code-opened",
-	onboarding_step_opened: "onboarding:step-opened",
-	onboarding_step_completed: "onboarding:step-completed",
-	onboarding_completed: "onboarding:completed",
 	postPush_emptyStateCtaClicked: "post-push:empty-state-cta-clicked",
 	postPush_toastCtaClicked: "post-push:toast-cta-clicked",
 	experiment_exposure: "experiment:exposure",
 	sliceName_pascalCaseError: "slice-name-error:pascal-case",
+	sharedOnboardingGuide_stepCompleted: "shared-onboarding-guide:step-completed",
```

Comment:
❓ Actually, why we don't re-use existing event? It's a demand from Product?

Reply:
yes, I was discussing that with Come, now the same event will be also triggered in PB so we want a new one with changed name and one property added